client/src/containers/Profile/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ProfileView from '../../views/Profile';
+import fetchProfileData from './fetchProfileData';
+
+function Profile(props) {
+  const [userData, setUserData] = useState({});
+  useEffect(() => {
+    const userId = props.match.params;
+    fetchProfileData(userId, setUserData);
+  // eslint-disable-next-line react/destructuring-assignment
```

Comment:
why?

Reply:
// eslint-disable-next-line react/destructuring-assignment
  }, [props.match.params.userId]); 
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md
eslint here expects us to actually destructure the props beforehand rather than use props as 'props.variable'. Hence disabled it because destructuring till props.match.params.userId level deep seemed like an overkill.